stable/rabbitmq/README.md

```diff
@@ -77,10 +77,18 @@ The following table lists the configurable parameters of the RabbitMQ chart and
 | `rabbitmq.configuration`             | Required cluster configuration                   | See values.yaml                                         |
 | `rabbitmq.extraConfiguration`        | Extra configuration to add to rabbitmq.conf      | See values.yaml                                         |
 | `rabbitmq.advancedConfiguration`     | Extra configuration (in classic format) to add to advanced.config    | See values.yaml                                         |
+| `rabbitmq.ssl.enabled`                  | Enable ssl support to rabbitmq |                     |
+| `rabbitmq.ssl.failIfNoPeerCert`     | `true`  | When set to true, TLS connection will be rejected if client fails to provide a certificate |
+| `rabbitmq.ssl.sslOptionsVerify`     | `verify_peer`  | Should [peer verification](https://www.rabbitmq.com/ssl.html#peer-verification) be enabled? |
+| `rabbitmq.ssl.caCertificate`     | Ca certificate  | Certificate Authority (CA) bundle content |
+| `rabbitmq.ssl.serverCertificate`     | Server certificate  | Server certificate content |
+| `rabbitmq.ssl.serverKey`     | Server Key  | Server private key content |
```

Comment:
I'd rename all these parameters to use "tls" instead:

- `rabbitmq.ssl.enabled`  -> `rabbitmq.tls.enabled`
- `rabbitmq.ssl.failIfNoPeerCert` -> `rabbitmq.tls.failIfNoPeerCert`
- ...

Reply:
all renamed